packages/dd-trace/test/tracer.spec.js

```diff
@@ -89,38 +91,25 @@ describe('Tracer', () => {
     })
 
     describe('_dd.base_service', () => {
-      let genSpan
-
       it('should be set when tracer.trace service mismatches configured service', () => {
-        tracer.trace('name', { service: 'custom' }, span => {
-          genSpan = span
-        })
-        const tags = genSpan.context()._tags
-        expect(genSpan).to.be.instanceof(Span)
-        expect(tags).to.include({
-          [BASE_SERVICE]: 'service',
-          [SERVICE_NAME]: 'custom'
-        })
+        tracer.trace('name', { service: 'custom' }, () => {})
+        const trace = tracer._exporter.export.getCall(0).args[0][0]
+        expect(trace).to.have.property(EXPORT_SERVICE_NAME, 'custom')
+        expect(trace.meta).to.have.property(BASE_SERVICE, 'service')
       })
 
       it('should not be set when tracer.trace service is not supplied', () => {
-        tracer.trace('name', {}, span => {
-          genSpan = span
-        })
-        const tags = genSpan.context()._tags
-        expect(genSpan).to.be.instanceof(Span)
-        expect(tags).to.have.property(SERVICE_NAME, 'service')
-        expect(tags).to.not.have.property(BASE_SERVICE)
+        tracer.trace('name', {}, () => {})
+        const trace = tracer._exporter.export.getCall(0).args[0][0]
+        expect(trace).to.have.property(EXPORT_SERVICE_NAME, 'service')
+        expect(trace.meta).to.not.have.property(BASE_SERVICE)
       })
 
       it('should be set when tracer.trace service matched configured service', () => {
```

Comment:
```suggestion
      it('should not be set when tracer.trace service matched configured service', () => {
```
As you mentioned this test makes no sense with that missing `not` 🤦